### PR TITLE
Add samsungpay (again)

### PR DESCRIPTION
**Pull request by ct-clearhaus**
Ref. #97, #100, and #102 🙂

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -667,6 +667,53 @@ object][ApplePay-PaymentToken] for more information.
   strongly authenticated (SCA in PSD2).
 </p>
 
+##### Method: `samsungpay`
+
+Samsung Pay provides the payment details as either JWE or JWE/JWS JOSE objects.
+Only JWE/JWS is accepted.
```
Comment (mt-clearhaus) [inline source/index.html.md:673]:
```suggestion
Only the latter is accepted.
```

:golfing:

File: source/index.html.md [hunk 2]
```diff
@@ -667,6 +667,53 @@ object][ApplePay-PaymentToken] for more information.
...
+
+The signed JWE must currently be encrypted by Samsung using Clearhaus' RSA
+public key.  This must be provided with Samsung Pay in a CSR. Contact Clearhaus
+to exchange a CSR.
```
Comment (mt-clearhaus) [inline source/index.html.md:677]:
```suggestion
The signed JWE must be encrypted by Samsung using Clearhaus' RSA public key.
This must be provided with Samsung Pay in a CSR. Contact Clearhaus to exchange
a CSR.
```

File: source/index.html.md [hunk 3]
```diff
@@ -667,6 +667,53 @@ object][ApplePay-PaymentToken] for more information.
...
+
+<p class="alert alert-info">
+<b>Notice:</b> Samsung pay can currently not be used for recurring transactions.
```
Comment (mt-clearhaus) [inline source/index.html.md:680]:
```suggestion
<b>Notice:</b> Samsung Pay can currently not be used for recurring transactions.
```

**Comment by ct-clearhaus** [comment]
No observable traction even though it has been in beta for years. Thus, closing this PR.

**Pull request closed without merging**